appium/webdriver/appium_connection.py

```diff
@@ -12,17 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from selenium.webdriver.remote.remote_connection import RemoteConnection
+from selenium.webdriver.remote.remote_connection import RemoteConnection, urllib3
```

Comment:
btw, should we import `urllib3` from Selenium side instead of `import urllib3`?

Reply:
yes. This is a third party library, which means that we should either explicitly include it into Pipfile (which might lead to version conflicts) or just use the one, which is bundled with Selenium (and that makes sense because we override methods of the class imported from Selenium)

Reply:
👍